tools-public/toolpad/resources/bundleSizeQueries.ts

```diff
@@ -2,9 +2,6 @@
 import axios from 'axios';
 
 function getMainBundleLabel(bundleId: string): string {
```

Comment:
I feel like by default we should use the `bundleId`. `@material-ui/core/Textarea` looks clearer than `TextareaAutosize` to me. I recall Sebastian wanted the `bundleId` to never change so we could always compare sizes with any previous build. While I admire Sebastian for the depth at which he's considering problems, I never felt that this was matching the use case for the tool, so we could remove this constraint. In any case, if we want to keep `bundleId` immutable, it still feel like we would have renamed rules, but still default to the bundle id as the label.